eskipfile/file.go

```diff
@@ -45,9 +45,16 @@ func Open(path string) (*Client, error) {
 	return &Client{routes}, nil
 }
 
+func (c Client) LoadAndParseAll() (routeInfos []*eskip.RouteInfo, err error) {
```

Comment:
'routeInfos' here is visible in the docs, due to LoadAndParseAll is public, info doesn't have plural form. Please, give it a better name.